superset/config.py

```diff
@@ -314,6 +314,14 @@ class CeleryConfig(object):
 # contain all the external tables
 CSV_TO_HIVE_UPLOAD_DIRECTORY = 'EXTERNAL_HIVE_TABLES/'
 
+# This contains the mappings from the tableschema types to HIVE types
+TABLESCHEMA_TO_HIVE_TYPES = {
```

Comment:
I think this probably belongs in `db_engine_spec`. I agree this isn't cut-and-dry as different versions of Hive support different types.

Reply:
Done